### DISCUSSION TITLE
Release of 5.3.0

### DISCUSSION THREAD

**Discussion by HyperBrain** [serverless-heaven/serverless-webpack#1539] [Team Posts]
If you don't have any objections, I will do the 5.3.0 release end of this week, most likely on Saturday. I'll let the guys in my former company run some deployment tests with master until then and check their feedback.
It would be great if you can also give or organize some feedback about the current quality of master.

**Comment by hassankhan** [comment]
No objections here, will report back once I've had a chance to test `master` against my projects.

**Comment by HyperBrain** [comment]
@designfrontier @hassankhan 5.3.0 should be finished soon - there are only 2 PR's more to go that are tagged.
As soon as we have them in I'll finalize the release notes and do the release. Maybe I'll update the dependencies prior to the NPM upload to have less security issues in.

**Comment by HyperBrain** [comment]
I'm away until Tuesday. Then I will check if the remaining 2 PR's are updated. If not, we'll schedule them for 5.4.0 or 5.3.1. Tuesday will definitely be the release date.

**Comment by hassankhan** [comment]
Awesome, I've updated the date on the 5.3.0 milestone as well.

**Comment by HyperBrain** [comment]
Release on npm has been done (as well as the release notes and release on GitHub).
Hopefully we won't have regressions when people update ;-)
